iOS/AreUDone/AreUDone/Service/CardService/CardLocalDataSource.swift

```diff
@@ -10,24 +10,81 @@ import RealmSwift
 
 protocol CardLocalDataSourceable {
   
-  func save(cards: Cards)
-  func loadCards(at dateString: String) -> Cards?
+  func save(cards: [Card])
+  func save(cardDetail: CardDetail)
+  
+  func updateCardDetail(for id: Int, content: String?, dueDate: String?)
+  
+//  func loadCards(at dateString: String) -> Cards?
+  func loadCards(at dateString: String, completionHandler: @escaping ((Cards?) -> Void))
+  func loadCardDetail(for cardId: Int, completionHandler: @escaping ((CardDetail?) -> Void))
+  
+  func deleteCard(for cardId: Int)
 }
 
 final class CardLocalDataSource: CardLocalDataSourceable {
   
   let realm = try! Realm()
   
-  func save(cards: Cards) {
-    try! realm.write {
-      realm.add(cards, update: .modified)
+  func save(cards: [Card]) {
+    realm.writeOnMain {
+      cards.forEach {
+        self.realm.add($0, update: .all)
+      }
     }
   }
   
+  func save(cardDetail: CardDetail) {
+    
+    realm.writeOnMain {
+      self.realm.add(cardDetail, update: .all)
```

Comment:
문서 찾아보니깐 데이터가 많이 바뀌었을 때는 .all 이, 데이터가 조금 바뀌었을 때는 .modified 가 유리하다는데 그 기준이 안 나와있어서 뭘 해야할지 좀 고민되네요..
https://realm.io/docs/swift/latest/api/Classes/Realm/UpdatePolicy.html#/s:10RealmSwift0A0C12UpdatePolicyO8modifiedyA2EmF
If few or no of the properties are changing this will be faster than .all and reduce how much data has to be written to the Realm file. If all of the properties are changing, it may be slower than .all (but will never result in more data being written).

Reply:
아 뭔가 PUT 하고 FETCH 이런 느낌인 것 같네요 ㅋㅋㅋㅋ
좀 더 찾아보고 기준이 잡히면 수정하도록 하겠습니다!